frankhn/questioner
# Ft rsvp meetup

#### What does this PR do?
this PR allows the user to respond to the meetup (confirm his attendance)
#### Description of Task to be completed?
the user clicks the RSVP button then it gets recorded in the database
#### How should this be manually tested?
clicking the RSVP button, and go to profile to check it the meetup you scheduled is there!!
#### Any background context you want to provide?
the API records the user id meetup ID in the database 
#### What are the relevant pivotal tracker stories?
users should be able to respond to a meetup question API ID: #163148873
#### Questions:
